src/stories/SearchBar/index.js

```diff
@@ -88,6 +97,19 @@ storiesOf('SearchBar', module)
     })
   )
   
+  .add(
+    'Recommended Searches',
+    withInfo({
+      text: `
+        Option for add recommended searches links
+        `,
+      inline: true,
+    })(() => {
+      return (
+        <SearchBar recommendedSearches={demoRecomendedRoutes}/>
```

Comment:
@ujovlado na implementaci ty featury je videt, ze by si to zaslouzilo neco jako 'clear search'. Predstava - krizek vpravo searchbaru. Jak pro Recommended searche v Jobech, tak i u ostatnich komponent. Ted to musis rucne smazat. Neni tam zadny clearovatko.

Reply:
Napr zakliknu 'My jobs'. Jak se potom dostanu zpet. Tim ze odmazu query? Predstavoval bych si to na jeden click. Podobne jako v react-selectu -> Clearable